src/main.ts

```diff
@@ -39,14 +24,14 @@ async function run(): Promise<void> {
   try {
     statusRequest = makeStatusRequest();
   }
-  catch (error) {
+  catch (error: any) {
     core.setFailed(`Error creating status request object: ${error.message}`);
     return;
   }
 
   try {   
     await octokit.repos.createStatus(statusRequest);
```

Comment:
You may want to try: [`octokit.rest.repos.createCommitStatus`](https://octokit.github.io/rest.js/v19#:~:text=octokit.rest.repos.createCommitStatus)